packages/react/src/themes/teams/components/Flex/utils.ts

```diff
@@ -0,0 +1,15 @@
+export const toFlexAlignment = propValue => {
```

Comment:
```suggestion
export const toFlexAlignment = (propValue: string) => { // ???
```

Can we define type here explicitly?